Polygon_mesh_processing/test/Polygon_mesh_processing/CMakeLists.txt

```diff
@@ -146,6 +146,11 @@ if(TARGET CGAL::Ceres_support AND TARGET CGAL::Eigen3_support)
 #  target_link_libraries( test_pmp_repair_self_intersections PRIVATE ceres )
 endif(TARGET CGAL::Ceres_support AND TARGET CGAL::Eigen3_support)
 
+if (TARGET CGAL::Eigen3_support)
+  create_single_source_cgal_program("test_geodesic_isolevel_refinement.cpp")
+  target_link_libraries( test_geodesic_isolevel_refinement PUBLIC CGAL::Eigen3_support)
```

Comment:
Where is Eigen needed?

Reply:
It's a dependency of the heat method package.